webapp/store/selectors/plugins.js

```diff
@@ -50,7 +48,8 @@ export const sortPluginMetadata = pluginMeta => {
 export const uniquePathNames = metadata => {
   const plugins = sortPluginMetadata(metadata);
   const paths = new Set();
-  return plugins.map(plugin => {
+  return plugins.map(_plugin => {
```

Comment:
Was getting some funky bugs in the tests as a result of the plugin being passed in getting mutated. Making a copy fixed it.